Assets/Plugins/LeapMotion/Modules/Hands/Scripts/HandBinder/HandBinderAutoRigger.cs

```diff
@@ -0,0 +1,208 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+namespace Leap.Unity.HandsModule {
+
+    public static class HandBinderAutoRigger {
+
+        /// <summary>
+        /// This function is used to search the HandBinder scipts children transforms to auto assign them for the user
+        /// </summary>
+        /// <param name="handBinder">The binder that the found transforms will get assigned too</param>
+        public static void AutoRig(HandBinder handBinder) {
+            handBinder.SetEditorPose = false;
```

Comment:
Why set to false to then set to true at the end? it is not used elsewhere in this function before being set to true always